src/ui/simulator/application/study.cpp

```diff
@@ -224,8 +224,7 @@ class JobOpenStudy final : public Toolbox::Jobs::Job
 
         // Postflight
         logs.info();
-        study->ensureDataAreAllInitialized();
-
+        study->areas.ensureDataIsInitialized(study->parameters, options.loadOnlyNeeded);
```

Comment:
Possible change of behavior here. Previously, `loadOnlyNeeded` was implicitly set to `false`. Now it depends on the context.